Hologram-Data-Availability/hda-web-server/main/hda/hda.cpp

```diff
@@ -0,0 +1,97 @@
+/*
+    Copyright 2020 Google LLC
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        https://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+#include "hda.h"
+#include <vector>
+#include <sstream>
+using namespace Pistache;
+using namespace std;
+
+HDAHandler::HDAHandler()
+    :server(Address(Ipv4::any(), Port(8000))) {
+        init();
+}
+
+HDAHandler::HDAHandler(Address Addr)
+    :server(Addr) {
+        init();
+}
+
+void HDAHandler::init() {
+    using namespace Rest;
+    server.init();
+    
+    Routes::Get(router, "/:system", 
+        Routes::bind(&HDAHandler::getDashboard, this));
+    Routes::Get(router, "/:system/:source/:date", 
+        Routes::bind(&HDAHandler::sendCommand, this));
+    Routes::Get(router, "/", 
+        Routes::bind(&HDAHandler::getLastRefreshed, this));
+
+    server.setHandler(router.handler());
+}
+
+void HDAHandler::start() {
+    server.serve();
+}
+
+void HDAHandler::getLastRefreshed(const Rest::Request& request, 
+    Http::ResponseWriter response) {
+    UNUSED(request);
+    response.send(Http::Code::Ok, "Wed May 19 15:46:11 2020");
```

Comment:
Add TODO here saying "will apply actual value here. Currently it is for prototyping purposes".